README.md

```diff
@@ -101,27 +64,20 @@ cp config.yaml.example config.yaml
 
 #### Use account and password
 
+The configuration file for chatapi-single can be found here: [chatapi-single](https://github.com/bytemate/chatapi-single#config)
```

Comment:
Should we add cahtapi-single to dockercompose ?

Reply:
I think so. It will be more user-friendly to deploy